### PR TITLE
gitAndTools.delta: 0.1.1 -> 0.2.0

### PR DESCRIPTION
**Pull request by zowoq**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @marsam @Ma27

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/delta/default.nix [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
   src = fetchFromGitHub {
     owner = "dandavison";
     repo = pname;
     rev = version;
-    sha256 = "1b5ap468d0gvgwkx6wqxvayzda2xw95lymd0kl38nq1fc0ica6hk";
+    sha256 = "15c7rsmiinnpxh12520jz3wr0j86vgzjq1ss5pf30fa7lcgicb32";
   };
 
   LLVM_CONFIG_PATH = "${llvmPackages.llvm}/bin/llvm-config";
   LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
```
Comment (marsam) [inline pkgs/applications/version-management/git-and-tools/delta/default.nix:24]: I think we can remove these

File: pkgs/applications/version-management/git-and-tools/delta/default.nix [hunk 2]
```diff
@@ -1,32 +1,44 @@
...
 
-  cargoSha256 = "07mjl751r9d88fnmnan0ip0m3vxqf51vq2y7k3g3yywcgasj9jgr";
+  cargoSha256 = "0lzz32qh80s4dxr0d4pg0qagkn549lr4vbqknl2l0cmlq1bvvq6g";
 
   nativeBuildInputs = [ installShellFiles ];
 
+  buildInputs = lib.optionals stdenv.isDarwin [ CoreFoundation libiconv Security ];
```
Comment (marsam) [inline pkgs/applications/version-management/git-and-tools/delta/default.nix:30]: is `CoreFoundation` needed? I can build it without it

Comment (zowoq) [reply pkgs/applications/version-management/git-and-tools/delta/default.nix]: No, not needed. I misread the missing framework error.

File: pkgs/applications/version-management/git-and-tools/delta/default.nix (hunk 1 above)
Comment (zowoq) [reply pkgs/applications/version-management/git-and-tools/delta/default.nix]: Removed.

**Comment by avdv** [comment]
Result of `nixpkgs-review pr 91977` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
<br>- bat-extras.batdiff
<br>- gitAndTools.delta
</details>

**Pull request merged**